app/models/claim.rb

```diff
@@ -431,8 +433,24 @@ def same_claimant?(other_claim)
     end
   end
 
+  def awaiting_provider_verification?
+    return false unless has_fe_policy?
+
+    return false if eligibility.verification.present?
+
+    if tasks.where(name: "matching_details").any?
+      tasks.where(name: "matching_details", passed: true).any?
+    else
+      Claim::MatchingAttributeFinder.new(self).matching_claims.empty?
+    end
```

Comment:
I know this is the logic described on the ticket but I wonder if what they _really_ want to know is if the provider verification email has been sent out and we're waiting on a response 🤔 if so we could move the logic from
`Policies::FurtherEducationPayments::AdminProviderVerificationTaskPresenter#verification_email_sent?` on to the eligibility then use that method here.

Reply:
I've asked on the ticket. If we change to using `verification_email_sent`, then the scope `awaiting_further_education_provider_verification` will have to change to check notes (I have a query ready that works).